src/platform/forms-system/src/js/definitions/email.js

```diff
@@ -11,7 +11,6 @@ export default function uiSchema(title = 'Email address') {
       required: 'Please enter an email address',
     },
     'ui:options': {
-      widgetClassNames: 'va-input-medium-large',
```

Comment:
Does this impact the way it looks on the review page?

Reply:
That class does shrink the email input on the review page as well

![Screen Shot 2020-03-16 at 11 22 16 AM](https://user-images.githubusercontent.com/136959/76778833-77b04a80-6778-11ea-88c2-b5d3ff0b3868.png)